pkg/drivers/http/driver.go

```diff
@@ -152,7 +152,7 @@ func (drv *Driver) Open(ctx context.Context, params drivers.Params) (drivers.HTM
 
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
+	if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusInternalServerError {
```

Comment:
Why did you add this extra condition?

Reply:
This also came from my local fork, because of some testing I was doing. I removed it.

Side note: an option to add custom "retry codes" (like what scrapy offers) would be good. Some sites return perfectly valid html but respond with status 500. I usually want to consider a request as failed only if it returned 200 or 202.

Reply:
Yeah, sure, you can def think about it.